.github/workflows/e2e_core.yml

```diff
@@ -1,9 +1,26 @@
-name: E2E Nightly
+name: E2E build & run
 
 on:
-  schedule:
-    # Run every day at 23:00 UTC
-    - cron: '0 23 * * *'
+  workflow_call:
```

Comment:
I'm afraid you will have to add some comments here to describe this incantation (and the one below) ;)

Reply:
Done.